### DISCUSSION TITLE
FATAL YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key (103:9)

### DISCUSSION THREAD

**Discussion by zilisrikle** [YunYouJun/yunyoujun.github.io#139] [Q&A]
请问出现这个错误该怎么解决，检查了下没有漏写空格
FATAL YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key (103:9)

 100 | theme: yun
 101 | autocrlf = false
 102 | safecrlf = false
 103 |   deploy: 
---------------^
 104 |   type: git
 105 |   repo: https://github.com/zilisrikle/z ...

**Comment by YunYouJun** [comment]
type: git 前应该缩进。

以及请在相关项目下提问。
